### PR TITLE
Expose verbose arg in RunnableRails

### PR DESCRIPTION
**Pull request by d-mariano**
## Summary
* Expose `LLMRails` `verbose` argument in `RunnableRails`
* Add simple unit test

### CODE REVIEW

File: tests/test_runnable_rails.py [hunk 1]
```diff
@@ -51,6 +50,24 @@ def test_string_in_string_out():
     assert result == "Paris."
 
 
+def test_string_in_string_out_with_verbose_flag():
```
Comment (d-mariano) [inline tests/test_runnable_rails.py:53]: We could also test a bit deeper here and look for logs made or events stored in history when `verbose` is `True`. Perhaps we should also assert that `verbose` is `False` by default? I can even move the test further down the file where we test other optional arguments, if that is also desirable.

Comment (drazvan) [reply tests/test_runnable_rails.py:53]: I think this is enough. No need to test deeper.

**Review by drazvan — APPROVED** [review]
Thanks @d-mariano! 👍

**Comment by d-mariano** [comment]
> Thanks @d-mariano! 👍

My pleasure @drazvan!

I just resolved the merge conflict for `CHANGELOG.md`. Let me know if there is anything else that you need me to do!

**Review by drazvan — APPROVED** [review]
Looks good! 👍

**Pull request merged**